apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningService.kt

```diff
@@ -73,6 +73,19 @@ class BeregningService(
         }
     }
 
+    private fun hentBeregning(behandlingId: UUID): Beregning? {
+        logger.info("Henter beregning for behandlingId=$behandlingId")
+
+        return beregningRepository.hent(behandlingId)
+    }
+
+    private suspend fun Beregning?.berikeMedManueltBeregning(brukerTokenInfo: BrukerTokenInfo) =
+        this?.let { beregning ->
```

Comment:
Mulig dette bare er en preferansegreie men jeg synes at man her gjør alt for å skrive single expression returns via lambda wrappere som gjør koden litt tung å lese imo.
Hva er galt med en gammel if else nullsjekk før denne kalles i funksjonen over så slipper vi å ha en extension function på en nullable prop som den må sjekkes for :exploding_head:

Reply:
Valgte det fordi det brukes mer en ett sted - så var samling av logikk.

Det er nok preferanser her ja - men pga at det kalles fra to steder så tror jeg nok at min pref her er sånn det er :)